Misc/NEWS.d/next/Library/2021-12-08-19-15-03.bpo-46016.s9PuyF.rst

```diff
@@ -0,0 +1 @@
+- Adding F_DUP2FD_CLOEXEC flag from FreeBSD into the fcntl module.
```

Comment:
I would be nice to document the constant somewhere in Doc/library/fcntl.rst:
https://docs.python.org/dev/library/fcntl.html

You can also document the new constant in What's New in Python 3.11, add a new "fcntl" section to:
https://docs.python.org/dev/whatsnew/3.11.html#improved-modules